lessons/prework/AC3.2Prework.playground/Pages/2. Arrays and Dictionaries.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,46 @@
+//: [Previous](@previous)
+
+import Foundation
+
```

Comment:
While these are good and simple Array and Dictionary explanations, we're going to hold off on those collection types for a while. I have to think whether this is still valuable as a preview or if we should hold off on them and focus on something else here in the prework.